Makefile

```diff
@@ -128,14 +128,17 @@ format:
 	$(ISORT) $(PYTHON_FILES)
 
 
-# make sure that the code conforms to the style guide
-# The DJANGO_SETTINGS module must be made available to pylint
-# to support e.g. string model referencec (see
-# https://github.com/PyCQA/pylint-django#usage)
+# make sure that the code conforms to the style guide. Note that
+# - the DJANGO_SETTINGS module must be made available to pylint
+#   to support e.g. string model referencec (see
+#   https://github.com/PyCQA/pylint-django#usage)
+# - export of migrations for prometheus stats must be disabled,
+#   otherwise it's attempted to connect to the db during linting
+#   (which is not available)
 .PHONY: lint
 lint:
 	@echo "Run pylint..."
-	DJANGO_SETTINGS_MODULE=config.settings $(PYLINT) $(PYTHON_FILES)
+	DJANGO_SETTINGS_MODULE=config.settings PROMETHEUS_EXPORT_MIGRATIONS=false $(PYLINT) $(PYTHON_FILES)
```

Comment:
It should have any impact but simply as styling, I would use `False` instead of `false`  or `0`